ext/session/tests/bug60860.phpt

```diff
@@ -12,6 +12,12 @@ echo "ok\n";
 
 ?>
 --EXPECTF--
-Warning: session_start(): user session functions not defined in %s on line 3
+Warning: session_start(): user session functions not defined in %s on line %d
 
-Fatal error: session_start(): Failed to initialize storage module: user (path:%s) in %s on line 3
+Warning: session_start(): Cannot send session cache limiter - headers already sent (output started at %s:%d) in %s on line %d
```

Comment:
Ok, I wondered about this. Failing to initialize the session storage should probably just be a fatal error anyway, so I'll revert this change.